llama-index-integrations/vector_stores/llama-index-vector-stores-chroma/tests/test_chromadb.py

```diff
@@ -1,16 +1,12 @@
 import os
-from typing import Dict, List
+from typing import Dict, List, Generator
 
 import pytest
-from llama_index.core.schema import NodeRelationship, RelatedNodeInfo, TextNode
-from llama_index.core.vector_stores.types import VectorStoreQuery
-from llama_index.vector_stores.chroma import ChromaVectorStore
+from llama_index.schema import NodeRelationship, RelatedNodeInfo, TextNode
+from llama_index.vector_stores import ChromaVectorStore
+from llama_index.vector_stores.types import VectorStoreQuery
```

Comment:
these imports should be updated to work with v0.10.0, I think you likely just need to revert this section

Reply:
fixed it